relays/ethereum/src/cli.yml

```diff
@@ -6,22 +6,22 @@ subcommands:
     - eth-to-sub:
         about: Synchronize headers from Ethereum node to Substrate node.
         args:
-            - eth-host:
+            - eth-host: &eth-host
```

Comment:
What's the point of adding `&arg`?

Reply:
That's what Tomek has suggested [here](https://github.com/paritytech/parity-bridges-common/pull/177#discussion_r451415868). TLDR: to avoid duplicate args declarations.

Reply:
Ah, I missed that, thanks!